clients/interfaces/urlstream.go

```diff
@@ -12,28 +12,6 @@
  * the License.
  *******************************************************************************/
 
-package urlclient
+package interfaces
 
-import (
-	"testing"
-
-	"github.com/edgexfoundry/go-mod-core-contracts/clients/types"
-)
-
-func TestClientFactoryLocal(t *testing.T) {
-	actualClient := New(types.EndpointParams{UseRegistry: false}, mockEndpoint{})
-	_, isLocalClient := actualClient.(*localClient)
-
-	if !isLocalClient {
-		t.Fatalf("expected type %T, found %T", localClient{}, actualClient)
-	}
-}
-
-func TestClientFactoryRegistry(t *testing.T) {
-	actualClient := New(types.EndpointParams{UseRegistry: true}, mockEndpoint{})
-	_, isRegistryClient := actualClient.(*registryClient)
-
-	if !isRegistryClient {
-		t.Fatalf("expected type %T, found %T", registryClient{}, actualClient)
-	}
-}
+type URLStream string
```

Comment:
[I structured this this way because of this SO answer](https://stackoverflow.com/questions/36820663/type-alias-for-channel-in-go-works-strange)